aws/resource_aws_glue_crawler_test.go

```diff
@@ -1391,6 +1516,146 @@ resource "aws_glue_crawler" "test" {
 `, rName, rName, path1, path2)
 }
 
+func testAccGlueCrawlerConfig_CatalogTarget_TableResources(tables []string) string {
```

Comment:
Rather than creating a separate function that generates configuration, this can be accomplished within configurations that potentially need multiple tables by using `count` and `count.index` with the resource. e.g.

```hcl
resource "aws_glue_catalog_table" "test" {
  count = %d

  database_name = "${aws_glue_catalog_database.test.name}"
  name          = "%s${count.index}"
  table_type    = "EXTERNAL_TABLE"

  storage_descriptor {
    location      = "s3://${aws_s3_bucket.default.bucket}"
  }
}
```

The count would be the number to create and the table name would be `rName`. References can be simply:

```hcl
aws_glue_catalog_table.test[*].name
```